### ISSUE TITLE
Game handle is not a link and is not selectable

### ISSUE THREAD

**Issue by quimnuss** [guyver2/battlechess#29]
When I try to select the handle text it seems to refresh the element and deselects it.

Since there also isn't a link to the game, the handle has to be copied by hand to share

**Comment by quimnuss** [comment]
On Firefox.

**Comment by guyver2** [comment]
yes it does refresh the whole page every 3 seconds (to fetch new games if any). It's part of the barely working prototype. Thanks for filling out such UX enhancement tickets.

**Comment by guyver2** [comment]
> Since there also isn't a link to the game, the handle has to be copied by hand to share

Today there is not way to join a non public game using the UI. It'll come later.
